device/globals/game.gd

```diff
@@ -216,7 +246,7 @@ func spawn_action_menu(obj):
 		player.walk_stop(player.position)
 
 	var pos = get_viewport().get_mouse_position()
-	var am_pos = action_menu.check_clamp(pos, camera)
+	var am_pos = action_menu.check_clamp(pos)
```

Comment:
But it looks like we're already using this confusing naming convention. It's understandable from context though.